Ginger/Ginger/Actions/ActionEditPage.xaml

```diff
@@ -278,7 +279,35 @@
                             </Grid>
                         </Expander.Content>
                     </Expander>
-                    <Ginger:ucGrid x:Name="OutputValuesGrid" Grid.Row="1" Margin="0,10,0,0" Title="Output Value(s)" ShowTitle="Collapsed" />
+
+                    <Expander  x:Name="ActionLogConfigExpander" ExpandDirection="Down" IsExpanded="False" Grid.Row="1" Style="{StaticResource @ExpanderLightStyle}" Expanded="ActionLogConfigExpander_Expanded"  Collapsed="ActionLogConfigExpander_Collapsed">
+                        <Expander.Header>
+                            <Label x:Name="ActionLogConfigExpanderLabel" Style="{StaticResource @HorizontalExpanderLightLabelStyle}">
+                                <Label.Content>
```

Comment:
Controls names should start with lower case x

Reply:
Fixed